# Write a program that uses a try-except block to handle division by zero

def divide_num(num1, num2):
    try:
        return num1/num2
    except ZeroDivisionError as err:
        return f"Zero division Run time error : {err}"

print(divide_num(1,0))
print(divide_num(100,10))
